### ISSUE TITLE
Handle TracWiki Macros

### ISSUE THREAD

**Issue by danuker** [chevah/trac-to-github#5]
As per https://github.com/chevah/trac-to-github/issues/2#issuecomment-764700864:

See how to convert TracWiki Macros like - [[TitleIndex(Administrative/)]]

TracWiki macro [[PageOutline]] can be converted into the RST content directive.

----

Suggestion from Dan: In order for TitleIndex macros to be more useful, a script should run regularly (say, every time the wiki is updated) and update the file lists, instead of just once, at importing.

**Comment by adiroiban** [comment]
> Suggestion from Dan: In order for TitleIndex macros to be more useful, a script should run regularly (say, every time the wiki is updated) and update the file lists, instead of just once, at importing.

Yes. i was thinking of creating some helper python script, hosted inside the wiki repo to inject per-file  footers.

Note that the GitHub wiki supports a general sidebar and footer that is show on all pages.

In RST, you can use the RST comments to mark the start of the footer. The script can then check each file, remove the current footer and inject a new one.

The RST comment can look like this.

```

.. footer-start-marker

```

--------

For pageoutline, just a script that adds the page TOC at the very top.

Thanks
